lib/miner-info.js

```diff
@@ -23,7 +23,8 @@ async function rpc (method, ...params) {
     method: 'POST',
     headers: {
       'content-type': 'application/json',
-      accepts: 'application/json'
+      accepts: 'application/json',
+      authorization: 'Bearer Bz8B9c8hKIp+/IyePpgORexkDDq+8c9atapgGuudtQ0='
```

Comment:
I am not a fan of having this configuration deep in the source code. Can we move the value to a constant exported by `lib/constants.js`?

For example:

```js
export const RPC_AUTH = 'Bearer Bz8B9c8hKIp+/IyePpgORexkDDq+8c9atapgGuudtQ0='
```

Reply:
https://github.com/filecoin-station/spark/pull/70